examples/diesel-postgres/src/main.rs

```diff
@@ -0,0 +1,119 @@
+//! Run with
+//!
+//! ```not_rust
+//! cargo run -p example-diesel-postgres
```

Comment:
Do we need a command here to run migrations?

Reply:
Yes that was missing.
Rather then requiring a separate step there I just used `diesel_migration` to automatically run the migrations on application startup. 
See https://github.com/tokio-rs/axum/pull/1853/commits/0abd85cf9e79eb4fa0066c0f0ce2880d71042817

(That's not possible with diesel-async yet, so no change there)